Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationItem.m

```diff
@@ -111,8 +127,55 @@ - (UIColor *)contentColor {
     return BPKColor.gray300;
 }
 
+- (void)setIconColor {
+    if (self.iconView != nil) {
+        self.iconView.tintColor = [self contentColor];
```

Comment:
Can we use property syntax for properties i.e. `self.contentColor` not `[self contentColor]`?